### DISCUSSION TITLE
MailHog and Laragon

### DISCUSSION THREAD

**Discussion by jennydupuy** [leokhoa/laragon#367] [Q&A]
Hi,

I installed MailHog under Laragon (see : [https://gist.github.com/reanim8ed/601e8a229fdad9a6c0c78f2415217389](https://gist.github.com/reanim8ed/601e8a229fdad9a6c0c78f2415217389))
MailHog runs fine with Laragon, but how do I stop it automatically when I leave Laragon?

Thanks for help.

**Comment by Pen-y-Fan** [comment]
From the command line, you can check if MailHog is still running:

```shell
tasklist | findstr /i /c:"MailHog.exe"
```

The output will be similar to:

```text
MailHog.exe                   9360 Console                    1     11,188 K
```

To kill the process:

```shell
taskkill /im mailhog.exe /f
```

Which will output similar to:

```text
SUCCESS: The process "MailHog.exe" with PID 9360 has been terminated.
```

This is dependent on the process being started as an administrator, or your account, you may need elevated privileges to stop it.

**Reply by Pen-y-Fan** [reply]
To use task manager

1. sort by process **name**
2. **expand all** (under **view** dropdown)
3. look down the list for Laragon (even after Laragon is closed, it will be shown under background processes)
4. right-click **MailHog.exe**
5. select **End task**

![image](https://user-images.githubusercontent.com/40126936/211197074-66a95d67-f9f1-459a-a598-673252d3f9a5.png)

Personally, MaiHog takes up 0.1% of memory and very little process, so I just leave it running, even after I close Laragon.
